### PR TITLE
Add SQL Linter using Squawk

### PR DESCRIPTION
**Pull request by febrianyuwono**
Adding an integration to Squawk ([website](https://squawkhq.com/) | [repo](https://github.com/sbdchd/squawk)) so we can double check if there are any database migrations that would cause unexpected downtime.

### CODE REVIEW

File: src/SquawkLinter.php [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+      if ($line == '0') {
+        $line = 1;
+      } else if ($column != '0') {
+        $line = $line + $column - 1;
+      }
+
+      $messages[] = id(new ArcanistLintMessage())
+        ->setName($this->getLinterName())
+        ->setCode($rulename)
+        ->setPath(idx($item, 'source', $path))
```
Comment (jparise) [inline src/SquawkLinter.php:148]: Based on the example output above, it looks like this key should be `file`?
```suggestion
        ->setPath(idx($item, 'file', $path))
```

File: src/SquawkLinter.php [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+        ->setLine($line)
+        ->setChar(1)
```
Comment (jparise) [inline src/SquawkLinter.php:150]: I would have expecting `$column` to be used here to set the "character" offset, but it looks like it's being used to adjust the `$line` above?

Comment (febrianyuwono) [reply src/SquawkLinter.php:150]: the `$column` here is showing how many rows are being warned by the linter
for example you have drop column like this:
```
ALTER TABLE

```

Comment (febrianyuwono) [reply src/SquawkLinter.php:150]: the `$column` here is showing how many rows are being warned by the linter
for example you have drop column like this:
```sql
   1 ALTER TABLE
   2   table_name
   3 DROP COLUMN
   4   column_name;
   5 
   6 -- test some column with spaces
   7               ALTER TABLE table_name DROP COLUMN column_name;
   8 
   9 
  10 -- test column with comment
  11 /* bla3 */ ALTER TABLE table_name DROP column column_name;
```

the first one will have line 1 column 0
the second one will have line 5 column 2 (so I added line + column - 1)
the third one will have line 8 column 3 (8+3-1 = 10)
to show which line is the starting one (includes comment)

and the char offset is not shown in the linter, so it's put as one

File: README.md [hunk 3]
```diff
@@ -310,6 +311,18 @@ Lints OpenAPI documents using [Spectral](https://stoplight.io/spectral).
...
 
+### Squawk
+
+Lints SQL files using [Squawk](https://squawkhq.com/).
+
+```json
+{
+    "type": "squawk",
+    "include": "(\\.sql)",
+    "spectral.ruleset": ".squawk.toml"
```
[the remark this answers is not in the document]
Comment (febrianyuwono) [reply README.md]: thank you for noticing, yes it should be `squawk.config`, I forgot to change this after copying when writing the readme

File: src/SquawkLinter.php (hunk 1 above)
Comment (febrianyuwono) [reply src/SquawkLinter.php]: yes, correct as well, thank you!

**Review by jparise — APPROVED** [review]
Looks good. Thanks for the contribution!

**Pull request merged**